src/components/navigator/NavBar.jsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import './navigator.css'
+
+const NavBar = () => {
+
+
+
+  return(
```

Comment:
Thanks for doing this! It's been something I've been kind of wanting to do for awhile as well.